docs/content/about.md

```diff
@@ -13,6 +13,7 @@ The Flink CDC Connectors integrates Debezium as the engine to capture data chang
 | PostgreSQL | Database: 9.6, 10, 11, 12 <br/>JDBC Driver: 42.2.12|
 | MongoDB | Database: 3.6, 4.x, 5.0 <br/>MongoDB Driver: 4.3.1|
 | Oracle | Database: 11, 12, 19 <br/>Oracle Driver: 19.3.0.0|
+| Sqlserver | Database: 2017, 2019 <br/>JDBC Driver: 7.2.2.jre8|
```

Comment:
Could we test and then recommend a higher JDBC driver for all of connector includes mysql, postgreql, sqlserver so that users can use one driver in their cluster lib directory ?

Reply:
Tips: We use `8.0.27` for these connectors in e2e tests.

Reply:
> 
This is very help for user to use flink-cdc-connector more conveniently.